pkg/globalconfig/global_config.go

```diff
@@ -75,7 +74,8 @@ type GlobalConfig struct {
 	RouterMailpitHTTPSPort           string                      `yaml:"mailpit_https_port,omitempty"`
 	Messages                         MessagesConfig              `yaml:"messages,omitempty"`
 	RemoteConfig                     RemoteConfig                `yaml:"remote_config,omitempty"`
-	ProjectList                      map[string]*ProjectInfo     `yaml:"project_info"`
+	// ProjectList is retained to make upgrading easier and should be removed after a few releases
+	ProjectList map[string]*ProjectInfo `yaml:"project_info"`
```

Comment:
I didn't like keeping this in the type since it's no longer intended to be kept there, but this is how we'd do this if we don't have an intermediary legacy type.